test/TestTerminalProvider.ts

```diff
@@ -52,6 +52,8 @@ class TestTerminal implements vscode.Terminal {
         this._errFilePath = path.join(root, `.err${this._suffix}`);
         this._semaphorePath = path.join(root, `.sem${this._suffix}`);
     }
+    creationOptions: Readonly<vscode.TerminalOptions> | Readonly<vscode.ExtensionTerminalOptions>;
```

Comment:
This class will be removed soon in #1888; I think that will cause a merge conflict in this so just a heads up. :smile: